demo/src/javascripts/components/demos/arc.js

```diff
@@ -108,16 +111,14 @@ export default class ArcDemo extends Component {
     selectedFeature = selectedFeature || features[362];
 
     const {flows, centroid, name} = selectedFeature.properties;
-    const arcs = [];
-
-    for (let toId in flows) {
+    const arcs = Object.keys(flows).reduce((acc, toId) => {
       const f = features[toId];
-      arcs.push({
+      return [...acc, {
```

Comment:
It's not usually a problem except in data processing. N in this case can be up to the 3k range so it's reasonable to prioritize performance. If N goes to 10k+ there will be a quite visible freeze.